samza-core/src/main/java/org/apache/samza/processor/StreamProcessor.java

```diff
@@ -283,24 +283,24 @@ public void onContainerFailed(Throwable t) {
               LOGGER.warn("JobCoordinatorLatch was null. It is possible for some component to be waiting.");
             }
             containerException = t;
-            LOGGER.error("Container failed. Stopping the processor.", containerException);
+            LOGGER.error(String.format("Container: %s failed. Stopping the stream processor: %s.", container, processorId), containerException);
```

Comment:
"... failed with an exception. Stopping the stream processor: %s. Original Exception:"